audiomate/corpus/io/common_voice.py

```diff
@@ -1,63 +1,107 @@
+    "sl": "https://voice-prod-bundler-ee1969a6ce8178826482b88e843c335139bd3fb4.s3.amazonaws.com/cv-corpus-5-2020-06-22/sl.tar.gz",  # noqa
+    "sv-SE": "https://voice-prod-bundler-ee1969a6ce8178826482b88e843c335139bd3fb4.s3.amazonaws.com/cv-corpus-5-2020-06-22/sv-SE.tar.gz",  # noqa
+    "ta": "https://voice-prod-bundler-ee1969a6ce8178826482b88e843c335139bd3fb4.s3.amazonaws.com/cv-corpus-5-2020-06-22/ta.tar.gz",  # noqa
+    "tr": "https://voice-prod-bundler-ee1969a6ce8178826482b88e843c335139bd3fb4.s3.amazonaws.com/cv-corpus-5-2020-06-22/tr.tar.gz",  # noqa
+    "tt": "https://voice-prod-bundler-ee1969a6ce8178826482b88e843c335139bd3fb4.s3.amazonaws.com/cv-corpus-5-2020-06-22/tt.tar.gz",  # noqa
+    "uk": "https://voice-prod-bundler-ee1969a6ce8178826482b88e843c335139bd3fb4.s3.amazonaws.com/cv-corpus-5-2020-06-22/uk.tar.gz",  # noqa
+    "vi": "https://voice-prod-bundler-ee1969a6ce8178826482b88e843c335139bd3fb4.s3.amazonaws.com/cv-corpus-5-2020-06-22/vi.tar.gz",  # noqa
+    "vot": "https://voice-prod-bundler-ee1969a6ce8178826482b88e843c335139bd3fb4.s3.amazonaws.com/cv-corpus-5-2020-06-22/vot.tar.gz",  # noqa
+    "zh-CN": "https://voice-prod-bundler-ee1969a6ce8178826482b88e843c335139bd3fb4.s3.amazonaws.com/cv-corpus-5-2020-06-22/zh-CN.tar.gz",  # noqa
+    "zh-HK": "https://voice-prod-bundler-ee1969a6ce8178826482b88e843c335139bd3fb4.s3.amazonaws.com/cv-corpus-5-2020-06-22/zh-TW.tar.gz",  # noqa
+    "zh-TW": "https://voice-prod-bundler-ee1969a6ce8178826482b88e843c335139bd3fb4.s3.amazonaws.com/cv-corpus-5-2020-06-22/zh-TW.tar.gz",  # noqa
+}  # noqa
+
+
+# ==================================================================================================
 
 
 class CommonVoiceDownloader(downloader.ArchiveDownloader):
     """
     Downloader for the Common Voice Speech Dataset.
 
     Args:
-      lang (string): Available languages are
-        de|en|et|it|fr|cy|br|cv|tr|tt|ky|ga-IE|kab|ca|zh-TW|sl|nl|cnh|eo|fa
-        See Common Voice dataset Version on download page for the
+      lang (string): See Common Voice dataset Version on download page for the
         corresponding language to the shortcut
     num_threads (int): Number of threads to use for download files.
     """
 
-    def __init__(self, lang='de', num_threads=1):
-        url = DOWNLOAD_URLS[lang]
-
-        super(CommonVoiceDownloader, self).__init__(
-            url,
-            move_files_up=True,
-            num_threads=num_threads
-        )
+    def __init__(self, lang="de"):
+        if lang in DOWNLOAD_URLS.keys():
+            self.url = DOWNLOAD_URLS[lang]
+            super(CommonVoiceDownloader, self).__init__(
+                self.url, move_files_up=True, num_threads=1
+            )
+        else:
+            msg = "There is no common-voice URL present for language {}!"
+            raise ValueError(msg.format(lang))
 
     @classmethod
     def type(cls):
-        return 'common-voice'
+        return "common-voice"
```

Comment:
This gets automatically changed with _black_ formatter. I can change it again, if it's important for you. In the other formatting cases black shoudn't break your pipeline. But I would prefer black's style over (partly) custom styles because it's always the same and not changing across different project which also makes contributing easier, because i don't have to setup or change anything.

Reply:
We're using flake8 to enforce the code style. If you think that flake8 is not appropriate, either open an issue or a PR with an explanation why we should change it.